planner/core/planbuilder.go

```diff
@@ -4200,6 +4200,7 @@ func (b *PlanBuilder) buildLoadData(ctx context.Context, ld *ast.LoadDataStmt) (
 		OnDuplicate:        ld.OnDuplicate,
 		Path:               ld.Path,
 		Format:             ld.Format,
+		SchemaName:         ld.Table.Schema.O,
```

Comment:
Seems this field is unnecessary.

Reply:
it's used to write to LOAD DATA's progress table, there's a column of schema name

Reply:
There is already a field `Table` that has the `SchemaName`.

Reply:
https://github.com/pingcap/tidb/pull/41871/commits/d7533ab80e20007d75dc425d5dbb57bf43a9afa6